simulation/utils/simulation_pygame_utils.py

```diff
@@ -1,7 +1,12 @@
 import pygame
+import pymunk
+
 from pygame.color import THECOLORS
 
-from utils.simulation_pymunk_utils import SCREEN_HEIGHT
+from utils.simulation_pymunk_utils import SCREEN_HEIGHT, Color
+
+GOAL_OBJECT_COLOR = Color(51, 153, 255)
```

Comment:
Te color(51, 153, 255) nic nie mówią. Może enum z kolorami i dopiero je tutaj przypisać? Tak żeby na pierwszy rzut oka móc powiedzieć "A, mapa ma kolor łososiowy, a obiekt to amarant wymieszany z fuksją"